r_basics_transform_data/r_basics_transform_data.md

```diff
@@ -437,7 +447,9 @@ In the box below, write a `filter()` statement that returns a data frame contain
 
 ********
 
-<div class = "warning">
+<div class = "help">
+<b style="color: rgb(var(--color-highlight));">Troubleshooting help</b><br>
```

Comment:
This is definitely WAAAAAY too long to be a troubleshoot box. This isn't a quick snippet of info about troubleshooting, this is moreso like substantial educational content about how something works. Perhaps it makes sense to put this in another issue/pulled out into another PR, but I don't think this really qualifies as a highlight box-able situation at all.

Reply:
I changed this to a header that just says "Strings and Quotes" so it is emphasized, but doesn't use a box.